react/utils/services.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+export const getSessionChannel = async () => {
+  const sessionRoute = '/api/sessions?items=*'
+
+  try {
+    const {
+      data: {
+        namespaces: {
+          store: {
+            channel: { value: sc },
+          },
+        },
+      },
+    } = await axios.get(sessionRoute)
+
+    return sc
+  } catch (error) {
+    throw error
+  }
+}
```

Comment:
Using `axios` directly is discouraged.
You can use the `useRenderSession()` hook or use the `tenantInfo` query of the `vtex.tenant-graphql` app, you should get the sales channel under `extraContext`.

```
query {
  tenantInfo {
    title
    bindings {
      defaultLocale
      extraContext
    }
  }
}
```

Reply:
Hello, after adding useRenderSession() following the indicated steps of the `vtex.session-client` app, it indicates in one of the steps to execute `vtex setup --typings`
after this I run the application and it does not allow me to handle the session object since it has not included the typings
error obtained:

12:33:44.266 - error: Webpack for react@3.x builder finished in 322ms with errors:
[at-loader] ../../.../react/components/MinicartFreeshipping.tsx:74:45
    TS2339: Property 'namespaces' does not exist on type 'Session'.
  Property 'namespaces' does not exist on type 'SessionUnauthorized'. vtex.builder-hub@0.284.3

Any idea how to fix this? thank you